content/3_Silver/Intro_Sorted_Sets.mdx

```diff
@@ -392,6 +392,39 @@ pq.add(6);                      // [7, 6, 5]
 
 </JavaSection>
 
+<PySection>
+
+### [Python](https://docs.python.org/3/library/heapq.html)
+
+In Python, we delete and retrieve the element of **lowest** rather than highest priority.
+
+Note that Python's priority queue is not encapsulated; `heapq` operates on a provided list directly by turning it into a heap, then doing operations on the heap.
+
+<Warning>
+
+Because of a heap's structure, printing out `pq` will **not** print out the elements in sorted order in Python. The comments below are a **representation** of what the heap contains, **not** what the contents of `pq` actually are.
```

Comment:
```suggestion
Because of a heap's structure, printing out `pq` will **not** print out the elements in sorted order in Python; instead, it will print out the list. The comments below are a **representation** of what the heap contains, **not** what the contents of `pq` actually are.
```